conf/application.conf

```diff
@@ -1,9 +1,72 @@
-# This is the main configuration file for the application.
-# ~~~~~
+play.i18n.langs = [ "en" ]
 
-application.secret="this is not a secret as we are not using this"
-application.langs="en"
-application.global=global.GlobalWithFilters
+play.modules {
+  enabled += "org.sagebionetworks.bridge.play.modules.BridgeSpringContextModule"
+}
+
+play.http.requestHandler = "org.sagebionetworks.bridge.play.http.BridgeHttpRequestHandler"
+
+play.http.errorHandler = "org.sagebionetworks.bridge.play.http.BridgeErrorHandler"
+
+play.http.filters = "org.sagebionetworks.bridge.play.http.BridgeFilters"
+
+play.filters {
+
+  # Security headers filter configuration
+  headers {
+
+    # The X-Frame-Options header. If null, the header is not set.
+    frameOptions = "DENY"
+
+    # The X-XSS-Protection header. If null, the header is not set.
+    xssProtection = "1; mode=block"
+
+    # The X-Content-Type-Options header. If null, the header is not set.
+    contentTypeOptions = "nosniff"
+
+    # The X-Permitted-Cross-Domain-Policies header. If null, the header is not set.
+    permittedCrossDomainPolicies = "master-only"
```

Comment:
This security header is for Adobe Flash or PDF Reader.  The master policy file is usually crossdomain.xml at the root of the site.  Since we are not supporting Flash or PDF out of the server and we don't have crossdomain.xml, I'm gonna remove this security header.